contracts/ClaimsManagerWithKlerosArbitration.sol

```diff
@@ -136,18 +137,13 @@ contract ClaimsManagerWithKlerosArbitration is
         payable
         override
     {
-        KlerosArbitrationParameters
-            storage klerosArbitrationParameters = klerosArbitrationParametersHistory[
-                claimIndexToKlerosArbitrationParametersHistoryIndex[claimIndex]
-            ];
-        IArbitrator arbitrator = klerosArbitrationParameters.arbitrator;
-        require(
-            claimIndex ==
-                klerosArbitratorToDisputeIdToClaimIndex[address(arbitrator)][
-                    disputeId
-                ],
-            "Invalid claim-dispute pair"
-        );
+        (
+            IArbitrator arbitrator,
```

Comment:
Why can you make use of the `IArbitrator` interface and the compiler doesn't complain even if it's not defined in the imports? 😕